tests/test_model_functions.py

```diff
@@ -28,24 +28,18 @@ class TestModelFunctions():
     })
 
     @pytest.fixture
-    def model(scope='class'):
```

Comment:
Can we import OXMETA from the conftest file?

Reply:
Done